### DISCUSSION TITLE
Image position

### DISCUSSION THREAD

**Discussion by henrysheehy** [GitJournal/GitJournal#967] [Feature Requests]
Great project!

As I’d like to integrate this with my Hugo site, I’m interested in adding support for image positioning, namely:
```markdown
![title](filename#position)
```
The position value can be set `left`, `right`, `full`. 
Would this be possible?
Thanks!

**Comment by vHanda** [comment]
Moving this to the feature request section.

But honestly, since this seems like something hugo specific and is not part of the markdown spec, I doubt I'll land up supporting it.
